mozilla/blurts-server
# Add a "Contact us" link to the user menu

# References: 
Jira: MNTOR-2880
Figma: N/A




# Description

Adds a "Contact us" link for Plus users.

# Screenshot (if applicable)

![image](https://github.com/mozilla/blurts-server/assets/4251/b51b60ef-9721-4fe7-b83b-a6c937be27ce)

# How to test

Open the user menu with a Plus account - you should see the link. Open it with a non-Plus account, and you shouldn't.

# Checklist (Definition of Done)
